core/test/fixtures/fraggle-rock/reports/sample-flow-result.json

```diff
@@ -3467,17 +3467,13 @@
               "items": []
             }
           },
-          "password-inputs-can-be-pasted-into": {
-            "id": "password-inputs-can-be-pasted-into",
-            "title": "Allows users to paste into password fields",
-            "description": "Preventing password pasting undermines good security policy. [Learn more about user-friendly password fields](https://web.dev/password-inputs-can-be-pasted-into/).",
-            "score": 1,
-            "scoreDisplayMode": "binary",
-            "details": {
-              "type": "table",
-              "headings": [],
-              "items": []
-            }
+          "inputs-can-be-pasted-into": {
+            "id": "inputs-can-be-pasted-into",
+            "title": "Allows users to paste into input fields",
+            "description": "Preventing input pasting undermines good security policy (in the case of passwords) and is a UX anti-pattern (in the general case). [Learn more about user-friendly input fields](https://web.dev/password-inputs-can-be-pasted-into/).",
+            "score": null,
+            "scoreDisplayMode": "error",
+            "errorMessage": "Required Inputs gatherer did not run."
```

Comment:
@adamraine how should I update these FR sample artifacts? We may need a script to do this (like we have for legacy runner)

Reply:
`yarn update:flow-sample-json --rebaseline-artifacts Scripts `

Reply:
hmmm

<img width="944" alt="image" src="https://user-images.githubusercontent.com/4071474/186239726-6ca02239-55a4-4837-ba7c-eb9fafd36cd3.png">

Reply:
Yeah maybe the script needs updating/hardening for this specific use case, but the structure is already there.